### ISSUE TITLE
Auth handler returns nil at load time

### ISSUE THREAD

**Issue by appgurueu** [minetest/minetest#11956]
I am cleaning up the file system at load time based on whether the auth handler returns an entry for a given playername. Unfortunately the auth handler always returns nil at load time. This is not documented. I would expect it to either error if it is unable to determine whether there is an auth entry.

**Comment by Zughy** [comment]
@appgurueu do you think the best approach is to document it? Also, if you don't mind, do you have a snippet I can easily test?

**Comment by appgurueu** [comment]
> @appgurueu do you think the best approach is to document it?

I'd prefer it if the auth handler worked at load time.

> Also, if you don't mind, do you have a snippet I can easily test?

`init.lua`:
```lua
assert(minetest.get_auth_handler().get_auth("Zughy") == nil)
minetest.after(0, function()
     assert(minetest.get_auth_handler().get_auth("Zughy") ~= nil)
end)
```

then join as `Zughy` & restart the server

**Comment by Desour** [comment]
I'm just seeing, this issue was not addressed in #13240. This is not about `minetest.get_auth_handler()` returning nil, but about `get_auth` returning nil.
Reopening.

**Comment by appgurueu** [comment]
Yeah, the *handler* returns `nil` - `get_auth_handler` *does not return `nil`* (otherwise these calls would not be possible).
